src/router_core/route_tables.c

```diff
@@ -243,7 +243,6 @@ void qdr_route_table_setup_CT(qdr_core_t *core)
         core->routers_by_mask_bit       = NEW_PTR_ARRAY(qdr_node_t, qd_bitmask_width());
         core->control_links_by_mask_bit = NEW_PTR_ARRAY(qdr_link_t, qd_bitmask_width());
         core->rnode_conns_by_mask_bit   = NEW_PTR_ARRAY(qdr_connection_t, qd_bitmask_width());
-        core->vflow_links_by_mask_bit   = NEW_PTR_ARRAY(vflow_record_t, qd_bitmask_width());
```

Comment:
This vflow_links_by_mask_bit is being removed because it is not being used anymore ?

Reply:
Yes